examples/todo/imports/ui/menu/Menu.js

```diff
@@ -0,0 +1,48 @@
+import { Meteor } from 'meteor/meteor';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Drawer from 'material-ui/Drawer';
+import { default as ListIcon } from 'material-ui-icons/List';
```

Comment:
I don't understand, why you change a way of imports this component.
I watch here: https://youtu.be/zbY-a5aOhr0?t=14238 but doesn't be explained.